MetaMask/metamask-mobile
# fix: update default search engine to google

## **Description**

1. What is the reason for the change?

We have received several reports of phishing links being the top result within DuckDuckGo's search engine when searching for official web3 sites. This is a security risk to users and creates a poor user experience if the user cannot trust the search results.

[See example here.](https://github.com/user-attachments/assets/d512f54c-06be-4120-970a-1edee0484a32) (first result is a phishing link, shown above the official Uniswap.org website)

3. What is the improvement/solution?

Switch the default search engine to Google, which shows more reliable results.

## **Manual testing steps**

1. Go to the Explore tab
2. Run a search query using the mobile browser
3. Ensure that the Browser searches on Google instead of DuckDuckGo


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
